pipestat/reports.py

```diff
@@ -100,6 +110,32 @@ def __call__(
         self.create_index_html(navbar, self.create_footer())
         return self.index_html_path
 
+    def _create_copy_for_porting(self, parentpath, record_identifier):
+        """
+        Helper function that assists with copying images (pdfs)
+        Original images stay in their original location.
+        This does NOT move the original data.
+
+        :param parentpath: path of existing file
+        :param newpath: desired destination
+        :return: newpath if successful else returns parentpath
+        """
+        destination_dir = os.path.join(self.pipeline_reports, "resources")
+        full_name = os.path.basename(parentpath)
+
+        destinationfile = os.path.join(destination_dir, record_identifier)
+        destinationfile = os.path.join(destinationfile, full_name)
```

Comment:
you can join more then 2 files at the same time

Reply:
yes, this should be one line

Reply:
done